src/python/pants/backend/python/rules/pex_environment.py

```diff
@@ -55,6 +55,18 @@ def register_options(cls, register):
                 "`[python-setup]` to influence where interpreters are searched for."
             ),
         )
+        register(
+            "--verbosity",
+            advanced=True,
+            type=int,
+            default=0,
+            help=(
+                "Set the verbosity level of PEX debug logging. "
+                "The higher the number, the more logging, with 0 being disabled, 3 being "
+                "equivalent to debug-devel logging, and 9 being equivalent to "
+                "trace-level logging."
```

Comment:
See above about a proposal to remove this mapping. If we do that, we can simplify this to something like:

> Set the verbosity level of PEX logging, from 0 (no logging) up to 9 (max logging).

Reply:
Done.